modules/apps/site/site-cms-site-initializer/src/main/resources/META-INF/resources/js/main/FDSPropsTransformer/actions/multipleFilesUploadAction.ts

```diff
@@ -22,6 +23,7 @@ export default function multipleFilesUploadAction(
 		contentComponent: ({closeModal}: {closeModal: () => void}) =>
 			MultipleFilesUploadModalContent({
 				assetLibraries: data.assetLibraries,
+				baseAssetLibraryViewURL: data.baseAssetLibraryViewURL,
```

Comment:
Just a question: cannot this be replaced by `{...data, onModalClose: closeModal}`?